### PR TITLE
Fix doctest from utf8proc update.

### PR DESCRIPTION
**Pull request by fredrikekre**
Hearts are now smaller.

### CODE REVIEW

**Comment by JeffBezanson** [comment]
This character is still full-width for me ¯\_(ツ)_/¯

File: base/strings/unicode.jl [hunk 1]
```diff
@@ -221,7 +221,7 @@ julia> textwidth('α')
 1
 
 julia> textwidth('❤')
-2
+1
```
Comment (JeffBezanson) [inline base/strings/unicode.jl]: We should find another example of a fullwidth character.

**Comment by KristofferC** [comment]
Tangentially, does failing doctests not fail the build anymore?

**Review by JeffBezanson — APPROVED** [review]

**Comment by StefanKarpinski** [comment]
> does failing doctests not fail the build anymore?

@staticfloat ^

**Comment by fredrikekre** [comment]
They are only running on Travis Linux 64-bit. I'm guessing this failure got through while Travis was a bit wonky for some days.

File: base/strings/unicode.jl (hunk 1 above)
Comment (stevengj) [reply base/strings/unicode.jl]: `⏰` ([U+23f0](https://www.fileformat.info/info/unicode/char/23f0/index.htm)) is defined as a wide character by [UAX 11](http://unicode.org/reports/tr11/), so it should always have width 2.

Comment (stevengj) [reply base/strings/unicode.jl]: The `⛵` (U+2685) in the modified PR is also "wide", so that should be fine too.

**Pull request merged**